components/local-app/cmd/login.go

```diff
@@ -66,7 +66,8 @@ var loginCmd = &cobra.Command{
 
 		err = auth.SetToken(loginOpts.Host, token)
 		if err != nil {
-			slog.Warn("could not write token to keyring, storing in config file instead", "err", err)
+			slog.Debug("could not write token to keyring, storing in config file instead")
```

Comment:
Could be worth checking `slog.Level` here to omit duplicate output.

Reply:
Maybe add a common function